File: java/template-renderer/src/main/java/com/rbleggi/templaterenderer/utils/FileUtil.java
package com.rbleggi.templaterenderer.utils;

import java.io.File;
import java.io.FileOutputStream;
import java.io.IOException;

public class FileUtil {
    public static void saveToFile(String filename, byte[] content) {
        var file = new File(filename);
        try (var outputStream = new FileOutputStream(file)) {
            outputStream.write(content);
            System.out.println("File saved to: " + filename + "\n");
        } catch (IOException e) {
            System.err.println("Error saving file: " + e.getMessage());
        }
    }
}

File: kotlin/template-renderer/src/main/kotlin/com/rbleggi/templaterenderer/utils/FileUtil.kt
package com.rbleggi.templaterenderer.utils

import java.io.File
import java.io.FileOutputStream

object FileUtil {
    fun saveToFile(filename: String, content: ByteArray) {
        val file = File(filename)
        FileOutputStream(file).use { outputStream -> outputStream.write(content) }
        println("File saved to: $filename\n")
    }
}